src/components/analyzeTable.vue

```diff
@@ -37,17 +39,23 @@
             {{ entry.desc }}
           </td>
           <td class="number">
-            {{ entry.min }}
+            {{ entry.min | withDecimalPlaces(3) }}
           </td>
           <td class="number">
-            {{ entry.max }}
+            {{ entry.max | withDecimalPlaces(3) }}
           </td>
           <td class="number">
-            {{ entry.avg }}
+            {{ entry.avg | withDecimalPlaces(3) }}
           </td>
           <td class="unit">
             {{ entry.unit }}
           </td>
+          <td class="number">
+            {{ entry.agg | withDecimalPlaces(0) }}
```

Comment:
If you've gone through with the aggregate count can be `null`, you problaby need some fancy check here.